nbs/API/frontmatter.ipynb

```diff
@@ -17,8 +17,7 @@
    "source": [
```

Comment:
<p>Was it intended to remove <code>order: 9</code> frontmatter ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>